MacOSX/build-package.in

```diff
@@ -6,6 +6,17 @@
 # You need to have  the following from homebrew or macports or fink:
 # autoconf automake libtool pkg-config
 
+# If you want to compile with OpenSCToken/CryptoTokenKit set the following:
+#ENABLE_CRYPTOTOKENKIT="--disable-pcsc  --enable-cryptotokenkit"
+# When using CryptoTokenKit, code signing is required
+#SIGNING_IDENTITY=BA3CE53D402E3C75246557E2890F929F4A778DDE
+
+if test -z "$ENABLE_CRYPTOTOKENKIT"; then
+	MACOS_VERSION_MIN="10.10"
```

Comment:
You can use env variable directly MACOSX_DEPLOYMENT_TARGET=10.10 also Xcode will use same variable later
No need to pass clang compiler parameters later